gitlab.go

```diff
@@ -300,7 +302,7 @@ func (c *Client) Do(req *http.Request, v interface{}) (*Response, error) {
 
 	if v != nil {
 		if w, ok := v.(io.Writer); ok {
-			io.Copy(w, resp.Body)
+			_, err = io.Copy(w, resp.Body)
```

Comment:
Bonus points: Look at `gometalinter`. It runs all the linters available. Some, like `golint`, can be very verbose e.g. if documentation is missing. But doing `gometalinter | less` is something I always do when starting to contribute to a Go project. On one hand it gives me an idea on how clean the code is, and on the other it can catch small typos/errors for a first PR :)